crates/dex-stable/src/primitives.rs

```diff
@@ -32,15 +32,22 @@ pub const MAX_A_CHANGE: u32 = 10;
 pub const MAX_ADMIN_FEE: Number = 10_000_000_000; // 100%
 pub const MAX_SWAP_FEE: Number = 100_000_000; // 1%
 
-#[derive(Encode, Decode, Clone, Default, PartialEq, Eq, Debug, TypeInfo)]
-pub struct BasePool<CurrencyId, AccountId, BoundString> {
-    pub currency_ids: Vec<CurrencyId>,
+#[derive(CloneNoBound, PartialEqNoBound, EqNoBound, RuntimeDebugNoBound, TypeInfo, Encode, Decode, MaxEncodedLen)]
+#[codec(mel_bound(skip_type_params(PoolCurrencyLimit, PoolCurrencySymbolLimit)))]
```

Comment:
Can you add some details about the new macros either as a doc comment or in the PR description?

Reply:
Not sure what details you would have me add, the macros are documented upstream.

Reply:
https://github.com/paritytech/parity-scale-codec/blob/b0099e80b1f6f09ab5cffe27794a11e134403b49/src/lib.rs#L241-L245